internal/pkg/deploy/cloudformation/stack/transformers.go

```diff
@@ -81,6 +82,12 @@ func convertSidecar(s map[string]*manifest.SidecarConfig) ([]*template.SidecarOp
 
 	var sidecars []*template.SidecarOpts
 	for _, name := range keys {
+		var sidecarPortMappings []*template.PortMapping
+		for _, portConfig := range portMappings {
```

Comment:
Maybe think twice when it is O(n^2) unless we can't mitigate this.

Reply:
I think I can use map for portMappings and get rid of this looping. I will do that.